token/core/fabtoken/issuer.go

```diff
@@ -50,11 +54,14 @@ func (s *Service) Issue(issuerIdentity view.Identity, typ string, values []uint6
 		nil
 }
 
+// VerifyIssue checks if the outputs of an IssueAction match the passed tokenInfos
 func (s *Service) VerifyIssue(tr driver.IssueAction, tokenInfos [][]byte) error {
 	// TODO:
```

Comment:
Should we leave this todo?

Reply:
Yes - it will be addressed in a different PR where we finalize the interface functionality.